gnosis/safe/addresses.py

```diff
@@ -595,6 +595,14 @@
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 2362236, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 2362238, "1.3.0"),
     ],
+    EthereumNetwork.ZETACHAIN_ATHENS_TESTNET: [
+        ("0xfb1bffC9d739B8D520DaF37dF666da4C687191EA", 1962980, "1.3.0+L2"),
```

Comment:
@nick8319  I couldn't verify the deploy block using the blockexplorer https://zetachain-athens-3.blockscout.com/address/0xfb1bffC9d739B8D520DaF37dF666da4C687191EA 
How do you get this block?

Reply:
Hey @moisses89, unfortunately Blockscout doesn't index internal transactions on Zetachain Testnet that's why it doesn't recognize `safe-contracts` as contracts. But I have transaction hashes saved during `safe-contracts` deployment. Here they are:
`GnosisSafe`: https://zetachain-athens-3.blockscout.com/tx/0x614a1428afd37470db6836abe53f191b3cc4470fd08212e0b3706166fcc24b44?tab=index
`GnosisSafeL2`: https://zetachain-athens-3.blockscout.com/tx/0x2934f55bae338f3af78fd2f86d98b42fd82f517feb9008b99837aaa4237b66e2?tab=index
`GnosisSafeProxyFactory`: https://zetachain-athens-3.blockscout.com/tx/0xfc3ba1aea3dd297d63172c4429bc59a47f6ddb0a1525ac156a6e83b8d6734cb9?tab=index